zzzz_modular_occulus/code/game/gamemodes/storytellers/storyevent.dm

```diff
@@ -18,8 +18,5 @@
 	req_crew = 12	//Makes it so that at least 9 players must be playing in order to spawn
 	max_crew_diff_lower = 3
 
-/datum/storyevent/spacevine
```

Comment:
Could have commented this out or set it to FALSE tbh. No need to delete it.

Reply:
This is our override. By default it's set to false.